config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :users, only: %i[index show]
+  resources :posts, only: %i[index show]
+
```

Comment:
- [ ] As required for the project, kindly make sure when typing the command `rails routes | grep user` should yield URLs that correspond to the below wireframes. Right now, they are not matching because of the way you place then. In order to match, you should update your routes to this:
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
![Screenshot from 2023-04-08 14-06-29](https://user-images.githubusercontent.com/83042742/230736570-2c37b33f-fc03-44e8-979a-0cd8c978305c.png)